resources/views/livewire/all-projects.blade.php

```diff
@@ -0,0 +1,233 @@
+<x-slot name="title">
+    {{ __('Projects') }}
+</x-slot>
+
+<x-slot name="header">
+    <h1 id="browse-all-projects">
+        {{ __('Browse all projects') }}
+    </h1>
+</x-slot>
+
+<div>
+    <div role="alert" x-data="{ visible: false }" @add-flash-message.window="visible = true"
+        @clear-flash-message.window="visible = false"
+        @remove-flash-message.window="setTimeout(() => visible = false, 5000)">
+        <div x-show="visible" x-transition:leave.duration.500ms>
+            @if (session()->has('message'))
+                <x-hearth-alert type="success">
+                    {!! Str::markdown(session('message')) !!}
+                </x-hearth-alert>
+            @endif
+        </div>
+    </div>
+
+    <form class="stack" wire:submit.prevent="search">
+        <x-hearth-label for="searchQuery" :value="__('Search')" />
+        <div class="repel">
+            <x-hearth-input name="searchQuery" type="search" wire:model.defer="searchQuery" wire:search="search" />
+            <button>{{ __('Search') }}</button>
+        </div>
+    </form>
+
+    <div role="alert">
+        @if ($searchQuery)
+            <p class="h4">
+                {{ __(':count results for “:searchQuery', ['count' => $projects->total(), 'searchQuery' => $searchQuery]) }}
```

Comment:
Can you add a period at the end of this sentence?

Reply:
We'll want to use `trans_choice` to handle the plural forms: https://laravel.com/docs/9.x/localization#pluralization